### ISSUE TITLE
🐞 encountered an issue when using the flake configuration provided in the README

### ISSUE THREAD

**Issue by pmpaulino** [dagger/nix#10]
### Description
I encountered an issue when using the flake configuration provided in the README of this repository. The configuration resulted in an error when trying to include Dagger as a package in a Nix development environment. Specifically, the error was: "attribute 'dagger' missing".

### Steps to Reproduce
I used the flake configuration as given in the README, which is as follows:

```
{
  inputs = {
    nixpkgs.url = "nixpkgs/nixos-unstable";
    flake-utils.url = "github:numtide/flake-utils";
    dagger.url = "github:dagger/nix";
    dagger.inputs.nixpkgs.follows = "nixpkgs";
  };

  outputs = { self, nixpkgs, flake-utils, dagger, ... }:
    flake-utils.lib.eachDefaultSystem (system:
      let
        pkgs = nixpkgs.legacyPackages.${system};
      in {
        devShell = pkgs.mkShell {
          buildInputs = [ dagger.packages.dagger ];
        };
      });
}
```

Running nix develop with this configuration led to an error.

### Suggested Fix
The issue was resolved by modifying the line in the devShell to reference the Dagger package for each system architecture specifically:

```
buildInputs = [ dagger.packages.${system}.dagger ];
```

### Dagger version

dagger v0.9.4 (registry.dagger.io/engine) darwin/arm64

**Comment by gerhard** [comment]
WDYT @sagikazarmark ?

**Comment by sagikazarmark** [comment]
Seems fair. Let me submit a PR to improve the PR in general.

**Comment by gerhard** [comment]
Is this still worth doing @sagikazarmark ?

**Comment by sagikazarmark** [comment]
Yep, it's still valid.

**Comment by gerhard** [comment]
OK!

Do you still want to submit a PR?

**Comment by sagikazarmark** [comment]
Looks like it was already fixed in #15
